src/OpenLoco/OpenLoco.cpp

```diff
@@ -815,6 +822,108 @@ namespace OpenLoco
         call(0x00496A84, regs);
     }
 
+    static void autosaveReset()
+    {
+        _monthsSinceLastAutosave = 0;
+    }
+
+    static void autosaveClean()
+    {
+        try
+        {
+            auto autosaveDirectory = Environment::getPath(Environment::path_id::autosave);
+            if (fs::is_directory(autosaveDirectory))
+            {
+                std::vector<fs::path> autosaveFiles;
+
+                // Collect all the autosave files
+                for (auto& f : fs::directory_iterator(autosaveDirectory))
+                {
+                    if (f.is_regular_file())
+                    {
+                        auto& path = f.path();
+                        auto filename = path.filename().u8string();
+                        if (Utility::startsWith(filename, "autosave_") && Utility::endsWith(filename, ".sv5", true))
```

Comment:
Like the base game, we append `.SV5` elsewhere. Using lowercase here therefore leads to problems on case-sensitive file systems. We really should support both, but for now:
```suggestion
                        if (Utility::startsWith(filename, "autosave_") && Utility::endsWith(filename, ".SV5", true))
```

Reply:
This is a case-insensitive comparison. I am not sure why this would make any difference.

Reply:
Right. I hadn't realised it was case-insensitive. In that case, there should be no difference, indeed. Arguably, it might be better to use uppercase for consistency, but it's not an issue for me.